lib/elixir/test/elixir/keyword_test.exs

```diff
@@ -24,7 +24,7 @@ defmodule KeywordTest do
 
   test "implements (almost) all functions in Map" do
     assert Map.__info__(:functions) -- Keyword.__info__(:functions) ==
-           [from_struct: 1]
+           [deep_merge: 2, deep_merge: 3, from_struct: 1]
```

Comment:
as mentioned before, I know that this is less than ideal but I wanna get the general deep_merge through before tackling it for `Keyword`